GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -357,23 +376,23 @@ gdjs.HowlerSoundManager.prototype.getGlobalVolume = function () {
 
 gdjs.HowlerSoundManager.prototype.clearAll = function () {
   for (var i = 0; i < this._freeSounds.length; ++i) {
-    if (this._freeSounds[i]) this._freeSounds[i].unload();
+    if (this._freeSounds[i]) this._freeSounds[i].stop();
```

Comment:
Is stop enough? The idea with `unload` was that we would really destroy and free the audio resource.

Reply:
The idea is to not unload it and keep it cached/loaded unless the user explicitly unloads it.